bundles/ch.elexis.core/src/ch/elexis/core/services/IUserService.java

```diff
@@ -166,9 +166,9 @@ default public Set<IMandator> getExecutiveDoctorsWorkingFor(IUserGroup group) {
 	 * Checks if the given {@link IUser} has the specified {@link IRole}
 	 * 
 	 * @param user
-	 * @param roleId
+	 * @param lRoleIds
 	 * @return
 	 */
-	public boolean hasRole(IUser user, String roleId);
+	public boolean hasRole(IUser user, List<String> lRoleIds);
```

Comment:
Bitte wie in der anderen Methode des Interface `Set<String> userRoles` anstelle der `List` verwenden, und die java doc anpassen, das geprüft wird ob eine aus dem `Set` matched.